packages/wrangler/src/__tests__/secret.test.ts

```diff
@@ -95,6 +96,83 @@ describe("wrangler secret", () => {
 				expect(std.err).toMatchInlineSnapshot(`""`);
 			});
 
+			it("should create secret:bulk", async () => {
+				writeFileSync(
+					"secret.json",
+					JSON.stringify({
+						"secret-name-1": "secret_text",
+						"secret-name-2": "secret_text",
+					})
+				);
+
+				// User counter to pass different secrets to the request mock
+				const counter = 0;
+				setMockResponse(
+					`/accounts/:accountId/workers/scripts/:scriptName/secrets`,
+					"PUT",
+					([_url, accountId]) => {
+						expect(accountId).toEqual("some-account-id");
+						counter + 1;
+
+						return { name: `secret-name-${counter}`, type: "secret_text" };
```

Comment:
This will always return `secret-name-0` for the reasons above

Reply:
It doesn't, that is why the tests are passing with the assertions for secret-name-1 & secret-name-2